### DISCUSSION TITLE
YARP Api gateway

### DISCUSSION THREAD

**Discussion by sathishsuresh04** [meysamhadeli/booking-microservices#268] [Q&A]
Hi,
All services are configured with Api versioning(EndpointConfig.BaseApiPath), but in the api gateway Yarp settings, versioning was not included . How come the gateway handled the versioning and is it working as expected?

**Comment by meysamhadeli** [comment]
Hi.
Yarp easily handle it for us with this configuration in our appsettings `"match": {
          "path": "flight/{**catch-all}"}` and in catch-all we can pass our version for example `1 or 2`, and base on this specific version, call specific version of our endpoint in for example flight.

> For more information, you can check documentation of [Yarp](https://microsoft.github.io/reverse-proxy/articles/getting-started.html#configuration)
